src/poetry/console/commands/lock.py

```diff
@@ -37,11 +37,16 @@ def handle(self) -> int:
         )
 
         if self.option("check"):
-            return (
-                0
-                if self.poetry.locker.is_locked() and self.poetry.locker.is_fresh()
-                else 1
+            if self.poetry.locker.is_locked() and self.poetry.locker.is_fresh():
+                self.line("poetry.lock is consistent with pyproject.toml")
```

Comment:
I would add a punctuation mark at the end of the message.

Reply:
Done